campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md

```diff
@@ -12,8 +12,15 @@ This walks through how to programmatically import and view your campaigns _via A
 * Your account has Messaging and Campaign Management products enabled
 * Your account has 10dlcImportCampaigns product feature enabled
 * Your account has your CSP ID associated to it
+* Your API User has the Campaign Management role assigned
 * You have provisioned campaigns with The Campaign Registry (TCR) through your Campaign Service Provider (CSP) 
 
+## Important Notes
+
+* If you get a '403 Unauthorized' error response while making an HTTP Request to any of the Import endpoints, 
+you will need to reach out to the Implementation team to get the Campaign Management role assigned to your API User.
+* TCR currently has rate limits set on their HTTP REST endpoints. During high volume events, it is possible our APIs will be rate limited and respond with '429 Too Many Requests'.
```

Comment:
```suggestion
* TCR currently has rate limits set on their HTTP REST endpoints. If you are sending a lot of traffic, it is possible that you will hit a rate limit and the API will respond with '429 Too Many Requests'.
```

Reply:
The original sentence reads like the entire API is rate limited across all users. I'm guessing it's rate limited per user?

Reply:
Unfortunately it is across all users. TCR sees our APIs as a single user and rate limits our creds. We're hoping that they'll up our rate limits once we start seeing 429 issues

Reply:
aaah. Ok sounds good